### PR TITLE
chore: update vendorized libraries to the latest versions

### PR DESCRIPTION
**Pull request by tranchitella**
Update the following vendorized libraries:
- github.com/stretchr/testify@v1.7.1
- github.com/nats-io/nats.go@v1.14.0
- go.mongodb.org/mongo-driver@v1.9.0

Changelog: None

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

File: go.mod [hunk 1]
```diff
@@ -6,13 +6,13 @@ require (
 	github.com/gin-gonic/gin v1.7.7
 	github.com/mendersoftware/go-lib-micro v0.0.0-20220304124258-d8414885f1c3
 	github.com/nats-io/nats-server/v2 v2.6.2 // indirect
-	github.com/nats-io/nats.go v1.13.0
+	github.com/nats-io/nats.go v1.14.0
```
Comment (alfrunes) [inline go.mod:9]:
```suggestion
	github.com/nats-io/nats.go v1.15.0
```

File: go.mod [hunk 2]
```diff
@@ -6,13 +6,13 @@ require (
 	github.com/gin-gonic/gin v1.7.7
 	github.com/mendersoftware/go-lib-micro v0.0.0-20220304124258-d8414885f1c3
 	github.com/nats-io/nats-server/v2 v2.6.2 // indirect
```
Comment (alfrunes) [inline go.mod:8]:
```suggestion
	github.com/nats-io/nats-server/v2 v2.8.2 // indirect
```

**Review by alfrunes — CHANGES_REQUESTED** [review]
They just released nats v1.15.0 today, would you mind upgrading so we don't get another dependabot PR? :)

**Review by alfrunes — APPROVED** [review]

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Comment by oleorhagen** [comment]
@tranchitella I started the pipe, feel free to stop it and merge if it is unnecessary you think

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Comment by tranchitella** [comment]
> @tranchitella I started the pipe, feel free to stop it and merge if it is unnecessary you think

It was blocked by the error in deployments-enterprise, it should be good now.

**Pull request merged**